### ISSUE TITLE
Edit : Renamed as 1.12.2 Evaluation about what is working (to be more positive) Crash cause not found yet

### ISSUE THREAD

**Issue by alxcool** [Tropicraft/Tropicraft#283]
Hi! I feel I was a bit hard on your mod, judging without knowing. So I've tested, played it with a minimum of others mods for a little bit and this is amazingly fun to play around! Thk for the mod.

Actually there is not much description on how to obtain slice / shop version of Pineapple, or Cocoa (Using a sword). It took me quite a while to figure it out. Also, I should have started exploring this mod whit an encyclopedia at hand, it help quite a lot to have information on mod's items and what to do with them.

So let start this comment around what is working, divided by the encyclopedia's chapters.
The encyclopedia include new topic when you have the items in your inventory.
### Tropical food
So far I could read as small description of Pineapple, pineapple cubes, coconut, coconut chunks, frog legs, frog legs, cooked, grapefruit, lemon, lime, orange, fresh marlin, raw spotted eagle ray.
At first I had a weird bug, where pina colada could not be crafted using the hand grid (2x2). The 3x3 grid require to be totally filled for the recipe to work. as example, I've used to place the required items in any order followed by sticks or cobblestone, just to fill the holes. But this somehow fix by itself and I don't have to do as such anymore. I just write this to leave a note if this happen to someone else. Its like if for the shapeless recipe instead of using "null" as to leave empty someone just write a space.

### Tropical Drink
There is a description for EIH Mixer, bamboo mug, Pina colada. This in world mixing system work really nicely. I was able to produce black coffee, by placing my roasted bean along as a bucket of tropical-water and an empty mug, in that order on the eih mixer, and the recipe complete by itself.

### Tropics world
I see Azurite, Zircon, Eudialyte, Purified sand, Coral Sands, Green Sands, Volcanic Sands, Mineral Sands. So far, only the green sand is a requirement in order to seed or regrow see weeds. Coral sand is not required to place coral. but coral sometime is created over manganese, then drop as item occasionally.
Minerals sand have the same purpose as sandstone.
### The Tribes
Section title don't print correctly in the encyclopedia. I could read about Ashen Mask, Koa Hunter, Koa Fisherman. Note the information about koa was only available after completing the trimix quest.
### Tropical Tools
There is an entry for dagger, bamboo spear, bamboo fishing rod, water wand, tropicraft portal en (probably enchanter wand), and the fishing net. I was unable to use the fishing net. There is no action linked to right click , even when I have an empty bucket and a bucket filled with tropical water in my inventory. Left click create an "attack". Killing the fish is not the main goal, right? As when I use the fishing rod, I have to stay still until the fish is out of the water, Its funny to see, but I'm not sure if there is something missing here. Note when a fish die on a shore, if its not by the hand of a player, there is no drops.
### Armors
There is 4 entry here, but they all have the same text. Using a single entry could solve this.

### Scuba Gear
There is an entry for Scuba flippers, scuba chestplate Gear, Scuba googles, Dive computer, Pony bottle, Diving Regulator, Scuba Tank and Trimix Gas Mixture. I wish I could read an entry about any Scuba tank filled with trimix gas as I was unable to craft one.
Looks like there is a difference between enyclopedia description, about the  air compressor (note there is no entry for this one.) Air compressor tool-tips suggest he may be able to fill two tanks at once, but so far I was not able to place a second one at a time. Is it possible that there is a missing inventory slot for the trimix or the second tanks? There is no GUI, so I won't know. I even try to mix both of them using an Anvil, a Crafting table, but it won't work.
### Beach Accessories
There is an entry for beach floats and beach Umbrella. Note to have both in inventory, as crafting is not enough to create an entry.
### Tropical Flora
There is an entry for Reeds, Bamboo Shoot, coffee Bean, Coral, Kapok Leaves, Mahogany logs**, Lemon Sapling**, Orange Sapling, Lime sapling. (Should be grapefruit), Palm Wood. Description of both logs are the inverse. Having a sapling in inventory trigger a different one as entry. No entry for any other leaves of saplings. No entry for grapefruit sapling.
### Decorative
Entry for Palm planks, Bamboo chest, bamboo door, bamboo stick, bamboo ladder, tropicraft item frame, chunk O head, tiki torches. This section work fine.
### Tropics Flowers
Co Diffusa, Crocosmia, Orchid, Canna, Anemone, Anthurium Orange, Anthurium Red, Magic mushroom, Pathos, Acai ine, Croton, Dracaena, Fern, Foliage, Bromeliad, Flower Pot. There is no entry for the tall flower called Iris, and or in any other chapter.

### Shells
There is an entry for Solo shell, Froxshell, Pad Shell, Rubeshell, Starfish shell, Turtle Shell, Pearl black, Pearl White. Note, encyclopedia don't print correctly the name of Solo, Frox and Rube shell.
Note: After some time there is enough variation of the Frox shell to fill a double chest. Is this on purpose? If so that is a nice list of contributors^^

### Tropical Music
There is an entry for The Tribe, Tradewinds, Low Tide. No entry for Buried Treasure and for the 2 others I din't encounter them in-game yet.

### Miscellaneous
There is an entry for coconut Bombs, Fertilizer, poison frog skin, Iguana Scale, Iguana Leather, .
 Note, I was unable to craft any Bowl to mix poison, to craft dart and JEI have no suggested recipes, nor items in his list.
Note, I'm not sure if the cocoa bomb is working, even when there is a player name in the config whitelist.
 So thanks for reading, and keep the good work! I'l let you know If I eventually reproduce the crash I had at first!

---

## Issue Description
singleplayer 1.12.2
tropicraft-MC1.12.2-7.1.4.78
forge 2705

## Steps to Reproduce
While in a chair, the game craft after I've drank pina colada.

Tropicraft Version:    78
Forge Version:    2705

# End of this comment
Is it possible that tropicraft use a dimension id who conflict with tamaized voidcraft?
Note, I will try to reproduce this crash with less mods quite soon.
https://minecraft.curseforge.com/projects/voidcraft

## Crash log
https://paste.dimdev.org/eyimiginim.mccrash
PASTE YOUR CRASH LOG HERE

---- Minecraft Crash Report ----
// I feel sad now :(

Time: 2018-06-11 22:49:44 EDT
Description: Exception in server tick loop

java.lang.ClassCastException: tamaized.voidcraft.common.world.dim.thevoid.BiomeGenVoid cannot be cast to net.tropicraft.core.common.biome.BiomeTropicraft
    at net.tropicraft.core.common.dimension.ChunkProviderTropicraft.populate(ChunkProviderTropicraft.java:80)
    at net.minecraft.world.chunk.Chunk.populate(Chunk.java:1017)
    at net.minecraft.world.chunk.Chunk.populate(Chunk.java:997)
    at net.minecraft.world.gen.ChunkProviderServer.provideChunk(ChunkProviderServer.java:157)
    at net.minecraft.world.World.getChunkFromChunkCoords(World.java:309)
    at net.minecraft.world.World.getChunkFromBlockCoords(World.java:304)
    at net.minecraft.world.World.getBlockState(World.java:910)
    at net.tropicraft.core.common.dimension.TeleporterTropics.placeInExistingPortal(TeleporterTropics.java:110)
    at net.tropicraft.core.common.dimension.TeleporterTropics.placeInPortal(TeleporterTropics.java:66)
    at net.minecraft.world.Teleporter.placeEntity(Teleporter.java:436)
    at net.minecraft.server.management.PlayerList.transferEntityToWorld(PlayerList.java:720)
    at net.minecraft.server.management.PlayerList.transferPlayerToDimension(PlayerList.java:623)
    at net.minecraft.server.management.PlayerList.transferPlayerToDimension(PlayerList.java:610)
    at net.tropicraft.core.common.dimension.TropicraftWorldUtils.teleportPlayer(TropicraftWorldUtils.java:37)
    at net.tropicraft.core.common.event.MiscEvents.tickServer(MiscEvents.java:59)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler_780_MiscEvents_tickServer_ServerTickEvent.invoke(.dynamic)
    at net.minecraftforge.fml.common.eventhandler.ASMEventHandler.invoke(ASMEventHandler.java:90)
    at net.minecraftforge.fml.common.eventhandler.EventBus.post(EventBus.java:182)
    at net.minecraftforge.fml.common.FMLCommonHandler.onPostServerTick(FMLCommonHandler.java:265)
    at net.minecraft.server.MinecraftServer.tick(MinecraftServer.java:712)
    at net.minecraft.server.integrated.IntegratedServer.tick(IntegratedServer.java:185)
    at net.minecraft.server.MinecraftServer.run(MinecraftServer.java:526)
    at java.lang.Thread.run(Unknown Source)

A detailed walkthrough of the error, its code path and all known details is as follows:
---------------------------------------------------------------------------------------

-- System Details --
  Minecraft Version: 1.12.2
  Operating System: Windows 7 (amd64) version 6.1
  Java Version: 1.8.0_171, Oracle Corporation
  Java VM Version: Java HotSpot(TM) 64-Bit Server VM (mixed mode), Oracle Corporation
  Memory: 1578172152 bytes (1505 MB) / 3279572992 bytes (3127 MB) up to 3941859328 bytes (3759 MB)
  JVM Flags: 7 total; -Xmx3772m -Xms1536m -XX:+UseConcMarkSweepGC -XX:+CMSIncrementalMode -XX:-UseAdaptiveSizePolicy -Xmn128M -XX:HeapDumpPath=MojangTricksIntelDriversForPerformance_javaw.exe_minecraft.exe.heapdump
  IntCache: cache: 0, tcache: 0, allocated: 1, tallocated: 50
  FML: MCP 9.42 Powered by Forge 14.23.4.2705 104 mods loaded, 104 mods active
       States: 'U' = Unloaded 'L' = Loaded 'C' = Constructed 'H' = Pre-initialized 'I' = Initialized 'J' = Post-initialized 'A' = Available 'D' = Disabled 'E' = Errored
       
       | State               | ID                                | Version          | Source                                                 | Signature                                |
       |:------------------- |:--------------------------------- |:---------------- |:------------------------------------------------------ |:---------------------------------------- |
       | UCHIJAAAAAAAAAAAAAA | minecraft                         | 1.12.2           | minecraft.jar                                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | mcp                               | 9.42             | minecraft.jar                                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | FML                               | 8.0.99.99        | forge-1.12.2-14.23.4.2705.jar                          | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | UCHIJAAAAAAAAAAAAAA | forge                             | 14.23.4.2705     | forge-1.12.2-14.23.4.2705.jar                          | e3c3d50c7c986df74c645c0ac54639741c90a557 |
       | UCHIJAAAAAAAAAAAAAA | bedrockminer                      | 1.0.3            | 0+Bedrock+Miner+1.12.2+by+XxRexRaptorxX+(v.1.0.3).jar  | None                                     |
       | UCHIJAAAAAAAAAAAAAA | betterunderground                 | 1.1.6            | 0+betterunderground-1.1.6.jar                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | coralreef                         | 2.0              | 0+CoralReef-2.5-1.12.2.jar                             | None                                     |
       | UCHIJAAAAAAAAAAAAAA | jei                               | 4.9.2.196        | jei_1.12.2-4.9.2.196.jar                               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | easiervillagertrading             | 1.2              | 0+easiervillagertrading-1.12-forge14.21.1.2387-1.2.jar | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enablecheats                      | @MODVERSION@     | 0+EnableCheats-1.12.2-1.0.0.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | fastleafdecay                     | v14              | 0+FastLeafDecay-Mod-1.12.1.jar                         | None                                     |
       | UCHIJAAAAAAAAAAAAAA | foragecraft                       | 1.14.2           | 0+foragecraft-1.14.2.jar                               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | geolosys                          | 1.9.1            | 0+Geolosys-1.12.2-1.9.1.jar                            | None                                     |
       | UCHIJAAAAAAAAAAAAAA | keywizard                         | 1.12.2-1.5.3-b1  | 0+keywizard-1.12.2-1.5.3-b1.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | lucky                             | 7.5.0            | 0+Lucky-Block-Mod-1.12.zip                             | None                                     |
       | UCHIJAAAAAAAAAAAAAA | moreloottables                    | 0.1.0            | 0+moreloottables-1.12.0-0.1.0.jar                      | None                                     |
       | UCHIJAAAAAAAAAAAAAA | naturescompass                    | 1.5.1            | 0+NaturesCompass-1.12.2-1.5.1.jar                      | None                                     |
       | UCHIJAAAAAAAAAAAAAA | openglider                        | @VERSION@        | 0+OpenGlider-1.12.1-1.1.0.jar                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | biomesoplenty                     | 7.0.1.2391       | BiomesOPlenty-1.12.2-7.0.1.2391-universal.jar          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | redstoneflux                      | 2.0.2            | The+IO+RedstoneFlux-1.12-2.0.2.3-universal.jar         | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | cofhcore                          | 4.5.1            | The+CoFHCore-1.12.2-4.5.1.18-universal.jar             | None                                     |
       | UCHIJAAAAAAAAAAAAAA | cofhworld                         | 1.2.0            | The+CoFHWorld-1.12.2-1.2.0.5-universal.jar             | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | thermalfoundation                 | 2.5.0            | ThermalFoundation-1.12.2-2.5.0.19-universal.jar        | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | ore_stone_variants                | 2.13             | 0+ore_stone_variants-2.13.jar                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | regrowableleaves                  | 4.0.0            | 0+regrowableleaves-1.12.2-4.0.0.jar                    | None                                     |
       | UCHIJAAAAAAAAAAAAAA | endercore                         | 1.12.2-0.5.30    | EnderCore-1.12.2-0.5.30.jar                            | None                                     |
       | UCHIJAAAAAAAAAAAAAA | codechickenlib                    | 3.1.7.340        | The+CodeChickenLib-1.12.2-3.1.7.340-universal.jar      | f1850c39b2516232a2108a7bd84d1cb5df93b261 |
       | UCHIJAAAAAAAAAAAAAA | thermalexpansion                  | 5.5.0            | ThermalExpansion-1.12.2-5.5.0.29-universal.jar         | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | enderio                           | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | xreliquary                        | 1.12.2-1.3.4.763 | 0+Reliquary-1.12.2-1.3.4.763.jar                       | None                                     |
       | UCHIJAAAAAAAAAAAAAA | roguelike                         | 1.8.0            | 0+RoguelikeDungeons-1.12.2-1.8.0.jar                   | None                                     |
       | UCHIJAAAAAAAAAAAAAA | simplesponge                      | 3.6.1            | 0+SimpleSponge-1.12.2-3.6.1.jar                        | 4ffa87db52cf086d00ecc4853a929367b1c39b5c |
       | UCHIJAAAAAAAAAAAAAA | unloader                          | 1.1.0            | 0+unloader-1.1.0.jar                                   | None                                     |
       | UCHIJAAAAAAAAAAAAAA | vanillafix                        | 1.0.8+88         | 0+VanillaFix-1.0.8-88.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | aroma1997core                     | 2.0.0.0.b154     | Aroma1997Core-1.12.2-2.0.0.0.b154.jar                  | dfbfe4c473253d8c5652417689848f650b2cbe32 |
       | UCHIJAAAAAAAAAAAAAA | aromabackup                       | 2.1.1.3.b101     | AromaBackup-1.12.2-2.1.1.3.b101.jar                    | dfbfe4c473253d8c5652417689848f650b2cbe32 |
       | UCHIJAAAAAAAAAAAAAA | aromabackuprecovery               | 2.1.1.3.b101     | AromaBackup-1.12.2-2.1.1.3.b101.jar                    | dfbfe4c473253d8c5652417689848f650b2cbe32 |
       | UCHIJAAAAAAAAAAAAAA | familiarfauna                     | 1.0.10           | Bio+FamiliarFauna-1.12.2-1.0.10.jar                    | None                                     |
       | UCHIJAAAAAAAAAAAAAA | craftingtweaks                    | 8.1.9            | BlayTheNinth+CraftingTweaks_1.12.2-8.1.9.jar           | None                                     |
       | UCHIJAAAAAAAAAAAAAA | waystones                         | 4.0.43           | BlayTheNinth+Waystones_1.12.2-4.0.43.jar               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | recycler                          | 1.5.0            | Corail+recycler-1.5.0-1.12.jar                         | None                                     |
       | UCHIJAAAAAAAAAAAAAA | scanner                           | 1.1.4            | Corail+scanner-1.1.4-1.12.jar                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | farseek                           | 2.3              | Delvr+Farseek-1.12-2.3.jar                             | None                                     |
       | UCHIJAAAAAAAAAAAAAA | repose                            | 1.6              | Delvr+Repose-1.12-1.6.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | streams                           | 0.4.4            | Delvr+Streams-1.12-0.4.4.jar                           | None                                     |
       | UCHIJAAAAAAAAAAAAAA | dooglamooworlds                   | 1.1.2            | dooglamooworlds-1.12-1.1.2.jar                         | None                                     |
       | UCHIJAAAAAAAAAAAAAA | tropicraft                        | 7.1.4.78         | End+tropicraft-MC1.12.2-7.1.4.78.jar                   | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiointegrationtic             | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiobase                       | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderioconduits                   | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderioconduitsappliedenergistics | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderioconduitsopencomputers      | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderioconduitsrefinedstorage     | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiointegrationforestry        | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiointegrationticlate         | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | ftblib                            | 5.1.0.21         | Jaredlll08+FTBLib-5.1.0.21.jar                         | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiomachines                   | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderiopowertools                 | 5.0.26           | EnderIO-1.12.2-5.0.26.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | enderzoo                          | 1.12.1-1.4.0.49  | EnderZoo-1.12.1-1.4.0.49.jar                           | None                                     |
       | UCHIJAAAAAAAAAAAAAA | endreborn                         | 0.1.4            | EndReborn+[0.1.5].jar                                  | None                                     |
       | UCHIJAAAAAAAAAAAAAA | tickdynamic                       | 1.0.1            | Fireplace+TickDynamic-1.12.2-1.0.1.jar                 | fc6c96880539df54ba847abda8844dabfcef307e |
       | UCHIJAAAAAAAAAAAAAA | clayspawn                         | 7.1.0            | FiresClaySpawn-1.12.2-7.1.0.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomosfoundry                   | 1.3.13           | Giacomos-Foundry-Mod-1.12.2.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomos_bookshelf                | 1.2              | GiacomosBookshelf-1.12.2-1.2.jar                       | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomoscompass                   | 1.0.6            | GiacomosCompass-1.12.2-1.0.6.jar                       | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomos_fishing_net              | 1.6.1            | GiacomosFishingNet-1.12.2-1.6.1.jar                    | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomos_map_merging              | 1.0              | GiacomosMapMerging-1.12.2-1.1.jar                      | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomos_maps                     | 1.2.3            | GiacomosMaps-1.12.2-1.2.3.jar                          | None                                     |
       | UCHIJAAAAAAAAAAAAAA | giacomostravelogue                | 1.0.6            | GiacomosTravelogue-1.12.2-1.0.6.jar                    | None                                     |
       | UCHIJAAAAAAAAAAAAAA | bamboozled                        | 1.1.0            | Insomniakitten+Bamboozled-1.12-1.1.0.jar               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | tammodized                        | 0.15.5           | Insomniakitten+TamModized-1.12.1-0.15.5.jar            | None                                     |
       | UCHIJAAAAAAAAAAAAAA | voidcraft                         | 0.26.10          | Insomniakitten+VoidCraft-1.12-0.26.10.jar              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | wearablebackpacks                 | 3.1.2            | Insomniakitten+WearableBackpacks-1.12.2-3.1.2.jar      | None                                     |
       | UCHIJAAAAAAAAAAAAAA | crafttweaker                      | 4.1.8            | Jaredlll08+CraftTweaker2-1.12-4.1.8.jar                | None                                     |
       | UCHIJAAAAAAAAAAAAAA | base                              | 3.8.0            | Jaredlll08+base-1.12.2-3.8.0.jar                       | None                                     |
       | UCHIJAAAAAAAAAAAAAA | clumps                            | 2.0.0            | Jaredlll08+Clumps-3.0.0.jar                            | None                                     |
       | UCHIJAAAAAAAAAAAAAA | ctgui                             | 1.0.0            | Jaredlll08+CraftTweaker2-1.12-4.1.8.jar                | None                                     |
       | UCHIJAAAAAAAAAAAAAA | crafttweakerjei                   | 2.0.2            | Jaredlll08+CraftTweaker2-1.12-4.1.8.jar                | None                                     |
       | UCHIJAAAAAAAAAAAAAA | waim                              | 1.0.0            | Jaredlll08+WhatAmIMissing-1.0.0.jar                    | None                                     |
       | UCHIJAAAAAAAAAAAAAA | yabba                             | 1.1.2.8          | Jaredlll08+YABBA-1.1.2.8.jar                           | None                                     |
       | UCHIJAAAAAAAAAAAAAA | blockdrops                        | 1.4.0            | jei+blockdrops-1.12.2-1.4.0.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | jeivillagers                      | 1.0              | JEI+jeivillagers-1.12-1.0.2.jar                        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | jeresources                       | 0.8.7.41         | Jei+JustEnoughResources-1.12.2-0.8.7.41.jar            | None                                     |
       | UCHIJAAAAAAAAAAAAAA | jeiintegration                    | 1.5.1.36         | jeiintegration_1.12.2-1.5.1.36.jar                     | None                                     |
       | UCHIJAAAAAAAAAAAAAA | lunatriuscore                     | 1.2.0.42         | LunatriusCore-1.12.2.jar                               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | ingameinfoxml                     | 2.8.2.94         | Lun+InGame-Info-XML-Mod-1.12.2.jar                     | None                                     |
       | UCHIJAAAAAAAAAAAAAA | fastbench                         | 1.5.0            | Shadows-of-fire+FastWorkbench-1.12.2-1.5.0.jar         | None                                     |
       | UCHIJAAAAAAAAAAAAAA | placebo                           | 1.3.4            | Shadows-of-fire+Placebo-1.12.2-1.3.4.jar               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | magicalmap                        | 3.1.0            | Shadows-of-fire+Magical+Map-1.12.2-3.1.0.jar           | None                                     |
       | UCHIJAAAAAAAAAAAAAA | plants2                           | 2.8.6            | Shadows-of-fire+Plants-1.12.2-2.8.6.jar                | None                                     |
       | UCHIJAAAAAAAAAAAAAA | applecore                         | 3.1.1            | Squeek502+AppleCore-mc1.12.2-3.1.1.jar                 | None                                     |
       | UCHIJAAAAAAAAAAAAAA | appleskin                         | 1.0.9            | Squeek502+AppleSkin-mc1.12-1.0.9.jar                   | None                                     |
       | UCHIJAAAAAAAAAAAAAA | waila                             | 1.8.26           | Squeek502+Hwyla-1.8.26-B41_1.12.2.jar                  | None                                     |
       | UCHIJAAAAAAAAAAAAAA | solcarrot                         | 1.3.1            | Squeek502+solcarrot-1.12.2-1.3.3.jar                   | None                                     |
       | UCHIJAAAAAAAAAAAAAA | wailaharvestability               | 1.1.12           | Squeek502+WailaHarvestability-mc1.12-1.1.12.jar        | None                                     |
       | UCHIJAAAAAAAAAAAAAA | wawla                             | 2.5.257          | Squeek502+Wawla-1.12.2-2.5.257.jar                     | d476d1b22b218a10d845928d1665d45fce301b27 |
       | UCHIJAAAAAAAAAAAAAA | funkylocomotion                   | 1.0              | The+funky-locomotion-1.12.2-1.1.1.jar                  | None                                     |
       | UCHIJAAAAAAAAAAAAAA | redstonearsenal                   | 2.5.1            | The+RedstoneArsenal-1.12.2-2.5.1.13-universal.jar      | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | diethopper                        | 1.1              | The+RWTema+Diet-Hoppers-Mod-1.12.2.jar                 | None                                     |
       | UCHIJAAAAAAAAAAAAAA | stem                              | @VERSION@        | The+stem-1.12.2-0.1.4.jar                              | None                                     |
       | UCHIJAAAAAAAAAAAAAA | thermaldynamics                   | 2.5.1            | ThermalDynamics-1.12.2-2.5.1.14-universal.jar          | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | thermalinnovation                 | 0.3.0            | ThermalInnovation-1.12.2-0.3.0.7-universal.jar         | 8a6abf2cb9e141b866580d369ba6548732eff25f |
       | UCHIJAAAAAAAAAAAAAA | tombstone                         | 2.9.6            | tombstone-2.9.6-1.12.jar                               | None                                     |
       | UCHIJAAAAAAAAAAAAAA | xaeroworldmap                     | 1.2.4            | XaerosWorldMap_1.2.4_Forge_1.12.jar                    | None                                     |
  Loaded coremods (and transformers): LoadingPlugin (Fireplace+TickDynamic-1.12.2-1.0.1.jar)
                                        com.wildex999.patcher.TransformerPatcher
                                        com.wildex999.patcher.EntityInjector
                                      FarseekCoreMod (Delvr+Farseek-1.12-2.3.jar)
                                        farseek.core.FarseekClassTransformer
                                      AppleCore (Squeek502+AppleCore-mc1.12.2-3.1.1.jar)
                                        squeek.applecore.asm.TransformerModuleHandler
                                      CoreMod (Aroma1997Core-1.12.2-2.0.0.0.b154.jar)
                                        
                                      VanillaFixLoadingPlugin (0+VanillaFix-1.0.8-88.jar)
                                        
                                      EnderCorePlugin (EnderCore-1.12.2-0.5.30.jar)
                                        com.enderio.core.common.transform.EnderCoreTransformer
                                        com.enderio.core.common.transform.SimpleMixinPatcher
                                      RBLoadingPlugin (0+RealBench-1.12.2-1.3.3.jar)
                                        pw.prok.realbench.asm.RBTransformer
  GL info: ~~ERROR~~ RuntimeException: No OpenGL context found in the current thread.
  Ender IO: No known problems detected.
            
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
            !!!You are looking at the diagnostics information, not at the crash.       !!!
            !!!Scroll up until you see the line with '---- Minecraft Crash Report ----'!!!
            !!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!!
  Suspected Mods: Tropicraft (tropicraft)
  Profiler Position: N/A (disabled)
  Player Count: 1 / 8; [EntityPlayerMP['Al_Cool'/76290, l='Fallofdumbness', x=-41.00, y=65.00, z=246.00]]
  Type: Integrated Server (map_client.txt)
  Is Modded: Definitely; Client brand changed to 'fml,forge'
  Client Crashes Since Restart: 0
  Integrated Server Crashes Since Restart: 1

**Comment by alxcool** [comment]
No luck! Iv'e created a game with only 4 mods, and I was able to visit Void (id -2) nether (id-1), end (id 1) and tropicraft (id -127).
I've also read the latest log, but din't learn much else than a few cascading worldgen and some intern name who use minecraft: as prefix.

[latest.log](https://github.com/Tropicraft/Tropicraft/files/2104071/latest.log)

I may try a few mods together with tropicraft to learn If I'm able to create a crash but, so far this looks like working just fine!

question : Does tropicraft dimension id is always -127? or its given randomly?

**Comment by alxcool** [comment]
What does that supposed to mean? I have enabled my custom set of mods agains, then instead of creating a crash, a weird behavior happen...
[17:49:52] [main/ERROR] [net.minecraft.client.network.NetHandlerPlayClient]: Received invalid update packet for null tile entity at BlockPos{x=1014, y=54, z=65} with data: {maxHeight:10,x:1014,y:54,z:65,id:"minecraft:seaweed",height:2}

When I drink a pina colada while being on a chair , on a beach at the sunset, it teleport me to the tropicraft dimension, then after 2sec I'm send back to my last overworld position with a chat message

... You should drink a pina colada before teleporting!
[latest.log](https://github.com/Tropicraft/Tropicraft/files/2104132/latest.log)


Note that If I was'in drinking I would not be able to join tropicraft dimension! There is a mention about enderio config file, I've reset it but noticed no change.

**Comment by alxcool** [comment]
Note : First comment have been edited, please read it again!

**Comment by tterrag1098** [comment]
To unlock the pages on koa tribesmen you must open the encyclopedia while near them. Simply walking by them without opening it will not be enough.

Thanks for the notes on the few mismatched pages/descriptions. I'll fix those soon.

Poison and poison darts are currently not implemented.

**Comment by alxcool** [comment]
Thk for the comments, I've fixed my crash by updating a few mods. My most annoying one was a server's freeze who disconnect the player without telling the client. (Being able to break blocks while game on pause.) It stop when I've disabled both of those mods : https://minecraft.curseforge.com/projects/ore-stone-variants
https://minecraft.curseforge.com/projects/historicized-medicine

Fix, by mods update. Thk for your support I'l close this now.
